user_manual/talk/talk_basics.rst

```diff
@@ -237,7 +237,9 @@ Starting a screen share
 -----------------------
 
 You can click the monitor icon on your video stream to share your screen. Depending on your browser, you will get the option to share a monitor, an application window or a single browser tab.
+If video from your camera is also available, other participants will see it in a small presenter view next to the screen share.
```

Comment:
Should we mention also that you can hide it and open it again from a button ?

Reply:
Other participants can do it, so I think, not related to a user who's reading this manual